config/tests/samples/create/harness.go

```diff
@@ -278,34 +278,37 @@ func MaybeSkip(t *testing.T, name string, resources []*unstructured.Unstructured
 	if os.Getenv("E2E_GCP_TARGET") == "mock" {
 		for _, resource := range resources {
 			gvk := resource.GroupVersionKind()
-			switch gvk.GroupKind() {
-			case schema.GroupKind{Group: "certificatemanager.cnrm.cloud.google.com", Kind: "CertificateManagerCertificate"}:
-			case schema.GroupKind{Group: "certificatemanager.cnrm.cloud.google.com", Kind: "CertificateManagerCertificateMap"}:
-			case schema.GroupKind{Group: "certificatemanager.cnrm.cloud.google.com", Kind: "CertificateManagerDNSAuthorization"}:
-				// ok
 
+			supported := false
+
+			switch gvk.Group {
+			case "certificatemanager.cnrm.cloud.google.com":
+				supported = true
+			}
+
+			switch gvk.GroupKind() {
 			case schema.GroupKind{Group: "iam.cnrm.cloud.google.com", Kind: "IAMServiceAccount"}:
-				// ok
+				supported = true
 
 			case schema.GroupKind{Group: "networkservices.cnrm.cloud.google.com", Kind: "NetworkServicesMesh"}:
-				// ok
+				supported = true
 
 			case schema.GroupKind{Group: "privateca.cnrm.cloud.google.com", Kind: "PrivateCACAPool"}:
-				// ok
+				supported = true
 
 			case schema.GroupKind{Group: "secretmanager.cnrm.cloud.google.com", Kind: "SecretManagerSecret"}:
-				// ok
 			case schema.GroupKind{Group: "secretmanager.cnrm.cloud.google.com", Kind: "SecretManagerSecretVersion"}:
-				// ok
+				supported = true
 
 			case schema.GroupKind{Group: "", Kind: "Secret"}:
-				// ok
+				supported = true
 
 			case schema.GroupKind{Group: "serviceusage.cnrm.cloud.google.com", Kind: "Service"}:
```

Comment:
If we're removing default, why bother keeping in this case with no code in it? Seems less confusing to remove the "Service" case.

Reply:
Not sure I follow what you're saying here

Reply:
Oh I get it - I was thinking about fallthrough, but we don't have fallthrogh

Reply:
I think removing the ok comments makes the logic clearer, without implying fallthrough